pom.xml

```diff
@@ -691,7 +691,7 @@
       </dependency>
       <dependency>
         <groupId>org.geotools</groupId>
-        <artifactId>gt-wfs</artifactId>
+        <artifactId>gt-wfs-ng</artifactId>
```

Comment:
I'm not sure what are the differences between gt-wfs ang ng one but we need to do updates in https://github.com/geonetwork/core-geonetwork/blob/geotools-20.4-upgrade/workers/wfsfeature-harvester/src/main/java/org/fao/geonet/harvester/wfsfeatures/worker/WFSHarvesterExchangeState.java#L68 ?

Reply:
It behaves the same way. The old one was based on `AbstractDatastore` and that one has been removed and is not longer part of Geotools.